src/antsibull_docs/process_docs.py

```diff
@@ -40,6 +41,48 @@ def get_collection_namespaces(collection_names: Iterable[str]) -> dict[str, list
     return namespaces
 
 
+_CAPITAL_LETTERS = tuple("ABCDEFGHIJKLMNOPQRSTUVWXYZ")
+
+
+def _exc_to_string(exc: p.ValidationError, model_name: str) -> str:
+    def _display_error_loc(error: pydantic_core.ErrorDetails) -> str:
+        # We sort out all strings starting with an upper-case letter
+        # since pydantic sometimes inserts class names, for example:
+        #
+        #   entry_points -> main -> seealso -> 1 -> SeeAlsoPluginSchema -> extra
+        #
+        # Since our attribute names are all lower-case and our class
+        # names all start with an upper-case letter, this effectively
+        # removes all class names from the location sequence.
```

Comment:
Hmm, the reasoning doesn't work due to how `options` etc. work - the keys there are arbitrary strings that users provide. There exist modules whose option names are starting with upper-case letters :(

For example, amazon.aws.elb_application_lb has a (nested) option `listeners.DefaultActions[].TargetGroupName`.

Reply:
Fixed in adcc19fdda1feaad869a2ac594318fad21c6f8e0.